### PR TITLE
[Docs] Fix missing li tag in extension points

### LINKED ISSUE #11782: [Docs] Missing li tag in extension points
### Current State
The `<li>` tag is missing in the [Extensions Point](https://docs.meshery.io/extensibility#types-of-extension-points) section.

### Desired State
Add the `<li>` tag there as well.

image

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by MohdMusaiyab**
This PR fixes #11782 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9acc4754501d7f15904517422c833940b054cd33 at: https://66d8b4d1fd91203007f42d41--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ea01804037b40b29be63db24e220fd4d5f01aa73 at: https://66dd700421f96ad0179da092--meshery-docs-preview.netlify.app

File: docs/_sass/elements.scss [hunk 1]
```diff
@@ -122,11 +122,6 @@ li:has(> a[href*="/guides/tutorials"]) {
   margin-left: -2rem;
 }
 
-li:has(> a[href*="/extensions"]) {
-  list-style-type: none;
-  margin-left: -2rem;
-}
```
Comment (Yashsharma1911) [inline docs/_sass/elements.scss:128]: This will affect other place where we have a[href*="/extensions"], have you tested it is not affecting other places?

Comment (MohdMusaiyab) [reply docs/_sass/elements.scss:128]: I checked through the UI across the site and found out that it was not affecting any other place where a[href*="/extensions"] is used.

**Review by leecalcote — APPROVED** [review]
This styling is intentional. I'll tweak a little as we move forward with merging.

**Pull request merged**